modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -197,6 +201,7 @@ public void processOpts() {
             supportingFiles.add(new SupportingFile("module.mustache", "app", "Module.java"));
         }
         supportingFiles.add(new SupportingFile("openapiUtils.mustache", "app/openapitools", "OpenAPIUtils.java"));
+        supportingFiles.add(new SupportingFile("securityApiUtils.mustache", "app/openapitools", "SecurityAPIUtils.java"));
```

Comment:
Maybe this file should only be generated if the user is using security . Not sure if it's possible to check that here

Reply:
Indeed, I do not think this is easy to do. We would need to check all api calls and determine if at least one of them uses a access token security. It is much simpler to always output the file, even if not needed, and simply rely on the build process of the client app to discard this file if no other file imports it. I think this is good enough.